__tests__/app.test.js

```diff
@@ -128,6 +129,47 @@ describe("GET /api/articles", ()=>{
         })
     })
 
+
+describe("GET /api/articles/:article_id/comments", ()=>{
+    test('200: responds with an array of comments for the given article_id', ()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body})=>{
+            const {comments} = body
+            expect(comments).toBeSortedBy("created_at")
+            comments.forEach((comment)=>{
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: 1
+                })
+            })
+
+        })
+    })
+    test('404: responds with an error message for a not found path', ()=>{ return request(app)
+        .get("/api/articles/100/comments")
+        .expect(404)
+        .then(({body})=>{
+            expect(body.msg).toBe("not found")
+        })
+    })
+    test('200: responds with an empty array if the article_id exists but there are no comments with that article id', ()=>{
+        return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then(({body})=>{
+            expect(body.comments).toEqual([])
+        })
+    })
+
+    })
+
+
```

Comment:
Good tests so far! One more that is important to have is a 400 for a bad request, what If the user tries to make a request for `GET /api/articles/notanumber/comments` ?